src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -350,6 +387,15 @@ describe('processBulkUpload', () => {
 			getMockJobHelpers(),
 		);
 		const updatedBulkUpload = await loadBulkUpload(bulkUpload.id);
+
+		const { rows: applicationForms } = await db.sql<ApplicationForm>(
+			'applicationForms.selectAll',
+		);
+		const [applicationForm] = applicationForms;
```

Comment:
You should be able to destructure directly to `applicationForm` like so, without needing the second assignment:

```ts
const { rows: [applicationForm] } = await db.sql<ApplicationForm>(
	'applicationForms.selectAll',
);
```

At least, that lints just fine on my end, do you want to give that a try?